src/server/utilities/liteATIClickTracking/index.ts

```diff
@@ -0,0 +1,92 @@
+export default () => {
+  window.addEventListener('load', () => {
+    document.addEventListener('click', (event: MouseEvent) => {
+      const targetElement = event.target as HTMLElement;
+      if (targetElement?.tagName === 'A') {
+        event.stopPropagation();
+        event.preventDefault();
+
+        const atiURL = targetElement.getAttribute('data-lite-ati-tracking');
+
+        if (atiURL == null) {
+          return;
+        }
```

Comment:
Will this not prevent links from proceeding if that don't have the ATI data-attribute?